zowe/zowe-cli-secrets-for-kubernetes
# solve tough-cookie dependency issue

**What It Does**


Solves #32 

Updated outdated dependencies, bumped npm version to `9.8.1` and node to `18.18.2` in order to provide supported versions since 16 seems to have reached [EOL](https://endoflife.date/nodejs).

this fix for `tough-cookie` dependency will use the `@kubernetes/client-node` version `1.0.0-rc4`of the API in order to use the fetch implementation mentioned in their repo instead of the deprecated and now vulnerable package of `request` the version `0.20.0` has. Once `v1.0.0` is released we can bump that version to that, but for the time being this can be used as a temporary fix for a critical vulnerability.

**How to Test**


Run
- `npm install`
- `npm run build`
- `npm run package`

plugin should install and work just as before

**Review Checklist**
I certify that I have:


**Additional Comments**
